test/js/bun/ffi/ffi.test.fixture.receiver.c

```diff
@@ -153,6 +192,25 @@ static bool JSVALUE_IS_NUMBER(EncodedJSValue val) {
   return val.asInt64 & NumberTag;
 }
 
+static uint8_t GET_JSTYPE(EncodedJSValue val) {
```

Comment:
Can we keep the always_inline attributes? I benchmarked this at one point and it did improve the performance some nanoseconds

Reply:
They're declared always_inline, we just don't repeat the attribute in the definition: https://github.com/oven-sh/bun/blob/657f5b9f6a91136d6a4e1bd1df8bd416d41d62ab/src/bun.js/api/FFI.h#L175